package.json

```diff
@@ -36,6 +36,7 @@
     "got": "11.8.6",
     "jayson": "3.6.5",
     "jsonwebtoken": "9.0.0",
+    "merkle-patricia-tree": "^4.2.4",
```

Comment:
Will this still work without the "^"?  I remember in our group code review, we said we'd prefer not to have it.